app/lnd/index.js

```diff
@@ -1,33 +1,44 @@
 import config from './config'
 import lightning from './lib/lightning'
 import walletUnlocker from './lib/walletUnlocker'
+import isLndRunning from './lib/util'
 import subscribe from './subscribe'
 import methods from './methods'
 import walletUnlockerMethods from './walletUnlockerMethods'
 // use mainLog because lndLog is reserved for the lnd binary itself
 import { mainLog } from '../utils/log'
 
-const initLnd = callback => {
+const initLnd = () => {
   const lndConfig = config.lnd()
-  const lnd = lightning(lndConfig.lightningRpc, lndConfig.lightningHost)
+  try {
+    const lnd = lightning(lndConfig.lightningRpc, lndConfig.lightningHost)
+    const lndSubscribe = mainWindow => subscribe(mainWindow, lnd, mainLog)
+    const lndMethods = (event, msg, data) => methods(lnd, mainLog, event, msg, data)
 
-  const lndSubscribe = mainWindow => subscribe(mainWindow, lnd, mainLog)
-  const lndMethods = (event, msg, data) => methods(lnd, mainLog, event, msg, data)
-
-  callback(lndSubscribe, lndMethods)
+    return {
+      lndSubscribe,
+      lndMethods
+    }
+  } catch (e) {
+    throw e
+  }
```

Comment:
What's the purpose of this catch block?

Reply:
This was a relic from earlier testing of the PR. I have since moved the error handling of these methods to the places where they are called - where we now show an error message to the user and abort the app if there is a problem.

See https://github.com/mrfelton/zap-desktop/blob/6d5163ade40805688f0fac53cc6c99eb336e8242/app/zap.js#L95-L113

This can happen if for example, you already have an lnd process running on your system which is not fully functional (eg missing tls cert in the expected location)

I've removed these unneeded try/catch blocks now.